src/app/pages/system/ssh-keypairs/ssh-keypairs-form/ssh-keypairs-form.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component } from '@angular/core';
```

Comment:
Why are these "keypairs" going in System--aren't they for replications?  For that matter, why are we calling them "keypairs" and not "Known Hosts" or some other clearer term?

Reply:
Because Key pairs is not the same as Known Hosts, completely different things.

They are for replications but they could eventually be used by some other subsystem, so they are shared instead of repeating the same thing for all. (I would be OK if UI want to restrict it to replication screen for now and abstract it away).

Reply:
The problem is that this is being seen as SSH only.  There are other authentication methods, and we should plan for those being used eventually even if it is not possible now.  SSH is just a special case.  The "keypair" term does not suggest that this is really a way of defining a known remote host and the authentication for it.

Reply:
Known Remote Host is not how SSH authentication works, you should read on that a bit.

You seem to have terms confused.

Reply:
I am not using the term Known Hosts as it is used by SSH.  The mistake is in thinking that this should be SSH-specific.  Corral used the term Peers, which is also terrible, but perhaps that will clarify.  The idea is that there is a list of predefined systems with pre-entered authentication information.  It might be SSH, it could be something else, and we should not box ourselves into a corner by making it SSH-specific.

Reply:
This one specifically is exclusive for SSH authentication.